src/graphics/SVG/Radar/ObstacleRenderer.php

```diff
@@ -83,4 +84,26 @@ protected function objectToSvgNode(string $cls): SVGNode
 
         return $svg;
     }
+
+    protected function meshToSVGNode(string $class): SVGNode
+    {
+        $svg = new $class();
+        $svgPolygon = SVGPolygon::class;
+        foreach ($this->obstacle->getFaces() as $meshFace)
+        {
+            $mesh = new $svgPolygon(
+                $meshFace->getVertices(),
+            );
+            $svg->addChild($mesh);
+        }
+
+        if ($this->bzwAttributesEnabled)
+        {
+            $svg->setAttribute('bzw:type', (string)$this->obstacle->getObjectType());
+            //$svg->setAttribute('bzw:vertex', vsprintf('%.3g %.3g %.3g', $converter->getBzwPosition()));
+            //$svg->setAttribute('bzw:facesize', (string)$obstacle->getFaceSize());
+            //$svg->setAttribute('bzw:face', implode(' ', $obstacle->getFaces()));
+        }
```

Comment:
```suggestion
```

There's some duplication that exists between `ObstacleRenderer::objectToSvgNode()` and other renderers (cuz I didn't realize it). So let's just remove this from here and let `MeshRenderer` take care of it.